modules/decision/decision.py

```diff
@@ -0,0 +1,62 @@
+"""
+Creates decision for next action based on LiDAR detections and current odometry.
+"""
+
+import enum
+from .. import decision_command
+from .. import detections_and_odometry
```

Comment:
take a look at the convention for imports https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2253226033/Python+Style+Convention#Imports